src/robot_interface/models/mission/mission.py

```diff
@@ -0,0 +1,31 @@
+from dataclasses import dataclass, field
+from datetime import datetime
+from typing import Iterator, List, Optional, Union
+from uuid import UUID, uuid4
```

Comment:
In the folder `src/robot_interface/models` create a `__init__.py` and reexport the lower classes such that in other files that want to use 
Mission
Task
Step
can write:
`from robot_interface.models import Mission,Task,Step`

Reply:
or `from robot_interface.models.mission import Mission,Task,Step`